domene/dokumentdata/src/main/java/no/nav/foreldrepenger/melding/dokumentdata/DokumentHendelse.java

```diff
@@ -0,0 +1,222 @@
+package no.nav.foreldrepenger.melding.dokumentdata;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.JoinColumnOrFormula;
+import org.hibernate.annotations.JoinFormula;
+
+import no.nav.foreldrepenger.melding.kodeverk.BehandlingType;
+import no.nav.foreldrepenger.melding.kodeverk.FagsakYtelseType;
+import no.nav.vedtak.felles.jpa.BaseEntitet;
+
+@Entity(name = "DokumentHendelse")
+@Table(name = "DOKUMENT_HENDELSE")
+public class DokumentHendelse extends BaseEntitet {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "SEQ_DOKUMENT_HENDELSE")
+    private Long id;
+
+    @Column(name = "behandling_id")
+    private Long behandlingId;
+
+    @ManyToOne
+    @JoinColumn(name = "dokument_mal_navn")
+    private DokumentMalType dokumentMalType;
+
+    @ManyToOne
+    @JoinColumnOrFormula(column = @JoinColumn(name = "behandling_type", referencedColumnName = "kode", nullable = false))
+    @JoinColumnOrFormula(formula = @JoinFormula(referencedColumnName = "kodeverk", value = "'" + BehandlingType.DISCRIMINATOR + "'"))
+    private BehandlingType behandlingType;
+
+    @ManyToOne
+    @JoinColumnOrFormula(column = @JoinColumn(name = "ytelse_type", referencedColumnName = "kode", nullable = false))
+    @JoinColumnOrFormula(formula = @JoinFormula(referencedColumnName = "kodeverk", value = "'" + FagsakYtelseType.DISCRIMINATOR + "'"))
+    private FagsakYtelseType ytelseType;
+
+    @Column(name = "gjelder_vedtak")
+    private boolean gjelderVedtak;
```

Comment:
Boolean, ikke bruk primitive.

Reply:
Hvorfor det egentlig?:)

Reply:
Jeg har endret det, men hvis man bryr seg om verdien vil man sette den uansett? Og i andre tilfeller så introduserer man muligheten for null? Hva vinner man?